pipeline/filters/lua.md

```diff
@@ -180,7 +176,25 @@ pipeline:
 {% endtab %}
 {% endtabs %}
 
-#### Environment variable processing
+### Number Type
+
+Lua treats numbers as a double type.
+It means an integer field (e.g. IDs, log levels) will be converted to a double.
+To avoid type conversion, the `type_int_key` property is available.
+
+### Protected Mode
+
+Fluent Bit supports protected mode to prevent crashes if it executes an invalid Lua script.
+See also [Error Handling in Application Code](https://www.lua.org/pil/24.3.1.html).
+
+
+## Code Examples
+
+For functional examples of this interface, please refer to the code samples provided in the source code of the project located here:
```

Comment:
```suggestion
For functional examples of this interface, refer to the code samples provided in the
[source code](https://github.com/fluent/fluent-bit/tree/master/scripts) of the
Fluent Bit project.
```

Reply:
@lockewritesdocs Isn't redundant to mention the Fluent Bit project? this is the documentation for it.

Reply:
oh, I see you changed it in the next suggestion. Correct?

Reply:
Rather than have a separate line for the link, I incorporated it into the suggested change on lines 194-196.